Automattic/jetpack
# Social Previews: Add sidebar UI

Fixes #16195

#### Changes proposed in this Pull Request:

* adds a new panel into the Jetpack sidebar, see #16195 for task definition
* refactors icons from the `publicize` feature into a shared file with icons so they can be shared (and adds google icon)
* differences from the original design:
  - icons are single color, as we are reusing `@automattic/social-icons` and those don't have multi-colored variants
  - we only show three services (WP will be added later)
  - the preview button is slightly misaligned from icons - not sure what to do here. buttons come with their padding and if we remove if, the hover/focus would look wrong. Equally wrong is when we artificially just move the button to the left and hover/focus will then reveal the outline of the button to be super close to the edge of the sidebar

<img width="279" alt="Screenshot 2020-08-03 at 15 08 19" src="https://user-images.githubusercontent.com/156676/89185865-38ee2600-d59b-11ea-8e99-8bafb0358205.png">


#### Jetpack product discussion


#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* same setup as #16615 
* Open the sidebar and check if you can see the UI with three service icons and a button (that does nothing)

#### Proposed changelog entry for your changes:
none yet
